scripts/globals/status.lua

```diff
@@ -1842,23 +1842,23 @@ xi.mod =
 
 xi.immunity =
 {
-    NONE        = 0x00,
-    SLEEP       = 0x01,
-    GRAVITY     = 0x02,
-    BIND        = 0x04,
-    STUN        = 0x08,
-    SILENCE     = 0x10,   -- 16
-    PARALYZE    = 0x20,   -- 32
-    BLIND       = 0x40,   -- 64
-    SLOW        = 0x80,   -- 128
-    POISON      = 0x100,  -- 256
-    ELEGY       = 0x200,  -- 512
-    REQUIEM     = 0x400,  -- 1024
-    LIGHT_SLEEP = 0x800,  -- 2048
-    DARK_SLEEP  = 0x1000, -- 4096
-    ASPIR       = 0x2000, -- 8192
-    TERROR      = 0x4000, -- 16384
-    DISPEL      = 0x8000, -- 32768
+    NONE        = 0,
```

Comment:
Are we moving away from hex overall? I kind of like the enums like this, but whatever makes the most sense

Reply:
The functions that take xi.immunity values accept decimal values. This is why I shifted away from using ENUMs. No lua functions involving this enum accepted hex values as far as I could tell.